### DISCUSSION TITLE
[Help] Why there is not any prebuilt action in action tab like before ?

### DISCUSSION THREAD

**Discussion by nareshroy8520** [anasty17/mirror-leech-telegram-bot#1191] [General]
How to deploy the bot as there is nothing in the action tab like before ? Please Help.

![Screenshot 2022-10-28 093650](https://user-images.githubusercontent.com/96655209/198500251-c3a279bd-0e6d-4ca0-a844-ad270c85e402.jpg)

**Comment by junedkh** [comment]
we remove heroku support

**Comment by sidhant-khamankar** [comment]
Any guide for people who bought heroku or hostinger plan?

**Comment by anasty17** [comment]
No, and vps is cheaper then heroku. If u have any student on heroku then u can't deploy don't ask about heroku its done from my side
